src/Commands/HTTPS/InfoCommand.php

```diff
@@ -4,16 +4,19 @@
 
 use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
 use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Friends\RowsOfFieldsInterface;
```

Comment:
What is RowsOfFieldsInterface for?  I see this has existed since before this PR, but I'm not sure what it's for.  In this branch, many classes implement RowsOfFieldsInterface, but it is not used as a typehint, nor does it appear to be for dependency injection. Seems like it could be removed. It's not doing any harm, if you want it for some sort of documentation, or if you plan on using it as a typehint in the future. (Not sure that you'd ever have need to do that with a Command, though.)

Reply:
I think it used to have more of a purpose but now has become part habit and part documentation. Less useless code is better so I'll remove it.